wagtail/admin/templates/wagtailadmin/pages/index.html

```diff
@@ -86,7 +86,9 @@
 
                         // Post
                         $.post(url, {csrfmiddlewaretoken: CSRFToken}, function(){
-                            addMessage('success', '"' + $(movedElement).data('page-title') + '" has been moved successfully.')
+                            const text = `"${(movedElement).data('page-title')}" has been moved successfully.`;
+                            const event = new CustomEvent('w-messages:add', { detail: { status: 'success', text, clear:true, } });
+                            document.dispatchEvent(event);
```

Comment:
```suggestion
                            const text = `"${$(movedElement).data('page-title')}" has been moved successfully.`;
                            const event = new CustomEvent('w-messages:add', { detail: { status: 'success', text, clear: true } });
                            document.dispatchEvent(event);
```

Be careful here with the double `$` usage, one is needed for template strings and the other is needed to wrap the `movedElement` for jQuery.

Also be sure to validate this behaviour (go to a parent page and switch to ordering mode to test).

<img width="218" alt="Screenshot 2023-03-07 at 8 53 25 pm" src="https://user-images.githubusercontent.com/1396140/223402146-4786030c-bc17-4050-8034-3f9611f2d336.png">

Click 'sort menu order'